hook-example.py

```diff
@@ -73,7 +73,8 @@ def compute_labels(images):
         image_timestamp = image['timestamp'] # in iso format string
         image_bytes = base64.b64decode(image['image_b64']) # the bytes
         image = Image.open(StringIO.StringIO(image_bytes)) # a PIL image
-    labels = ['cat','dog','mouse']
+        image_labels = ['cat','dog','mouse']
+        labels.append({'timestamp': image_timestamp, 'labels':image_labels})
```

Comment:
@mdipierro we still want to have a single `labels` list for the `Event` labels. 

But we just need to include the full fidelity of the labeling results associated with each individual timestamp that was labeled for future use (even though ignoring for now) perhaps like:
```json
{
  "2017-04-18T02:07:44.636-0000": {
    "cat":{"probability":0.90, "polygon":{}}
  },
  "2017-04-18T02:07:47.123-0000": {
    "cat":{"probability":0.93, "polygon":{}}, 
    "dog":{"probability":0.88, "polygon":{}},
  }
}
```